core/data_category.js

```diff
@@ -472,3 +475,17 @@ Blockly.DataCategory.createValue = function(valueName, type, value) {
       '</value>';
   return valueField;
 };
+
+/**
+ * Construct a block separator. Add the separator to the given xmlList.
+ * @param {!Array.<!Element>} xmlList Array of XML block elements.
+ */
+Blockly.DataCategory.addSep = function(xmlList) {
+    var gap = 36;
+    var sepText = '<xml>' +
+        '<sep gap="' + gap + '"/>' +
+        '</xml>';
+    var sep = Blockly.Xml.textToDom(sepText).firstChild;
+    xmlList.push(sep);
+  }
```

Comment:
You've got an extra close-brace here that's causing travis failures.

Reply:
I tried, and it didn't work.

Reply:
Oh, now it's working.